tests/FSharp.Compiler.ComponentTests/Conformance/Types/RecordTypes/AnonymousRecords.fs

```diff
@@ -74,4 +74,17 @@ module NestedAnonRecds
 let x = {| abcd = {| ab = 4; cd = 1 |} |}
 """
         |> compile
-        |> shouldSucceed
\ No newline at end of file
+        |> shouldSucceed
+
+    [<Fact>]
+    let ``Wrong update syntax`` () =
+        Fsx """
+let f (r: {| A: int |}) =
+    { r with A = 1 }
+"""
+        |> ignoreWarnings
+        |> compile
+        |> shouldFail
+        |> withDiagnostics [
+            (Error 39, Line 2, Col 36, Line 2, Col 37, "The record label 'A' is not defined.")
```

Comment:
```suggestion
            (Error 39, Line 3, Col 14, Line 3, Col 15, "The record label 'A' is not defined.")
```

Reply:
@T-Gro Thanks, I've pushed the fix just before it. 🙂